scipy/stats/__init__.py

```diff
@@ -263,6 +263,7 @@
    brunnermunzel
    combine_pvalues
    jarque_bera
+   Ks_2sampResult
```

Comment:
for the other tests that return named tulples, we do not mention the output here. do you propose to change that?

Reply:
By having `mstats_basic` K-S functions call `stats` K-S functions, I no longer needed to expose that named tuple. 
Exposing them all in the API would require taking a look at the names and rationalizing them, making consistent, as that is not the case, from a quick perusal.